submission-scripts/script-templates/Makefile

```diff
@@ -86,7 +86,7 @@ STORM_VERSION_DEFAULT=1.0.1
 STORM_ZOOKEEPER_VERSION_DEFAULT=3.4.8
 ZOOKEEPER_VERSION_DEFAULT=3.4.8
 TACHYON_VERSION_DEFAULT=0.6.1
-ZEPPELIN_VERSION_DEFAULT=0.5.6
+ZEPPELIN_VERSION_DEFAULT=0.5.6-incubating
```

Comment:
Actually, lets leave this as 0.5.6 for this patch.

The "-incubating" will have fallout that probably needs to be fixed in multiple places.  Minimally ZEPPELIN_HOME and the jar files loaded in magpie/setup/magpie-setup-project-zeppelin.